src/tools/schema2kotlin.ts

```diff
@@ -116,6 +135,25 @@ abstract class Abstract${particleName} : ${this.opts.wasm ? 'WasmParticleImpl' :
 `;
   }
 
+  private getQueryType(connection: HandleConnectionSpec): string {
```

Comment:
return type: string? or `string | null`.

Reply:
I don't think this syntax is supported for return types in typescript
https://github.com/microsoft/TypeScript/issues/13184

Looks like the above was never fixed as the difference between `T | undefined` and `T | null` is ambiguous